src/audio/dai.c

```diff
@@ -275,8 +275,8 @@ static int dai_playback_params(struct comp_dev *dev, uint32_t period_bytes,
 
 	/* set up DMA configuration */
 	config->direction = DMA_DIR_MEM_TO_DEV;
-	config->src_width = sample_bytes(dd->frame_fmt);
-	config->dest_width = sample_bytes(dd->frame_fmt);
+	config->src_width = get_sample_bytes(dd->frame_fmt);
+	config->dest_width = config->src_width;
 	config->cyclic = 1;
```

Comment:
This is fine now, but please do ONE kind of change in a single patch. I would have added:

```
config->src_width = get_sample_bytes(dd->frame_fmt);
config->dest_width =  get_sample_bytes(dd->frame_fmt); 
```
And then later if you want:

```
 get_sample_bytes(dd->frame_fmt);
```

this is a small change and we can take it as it is. But please keep this in mind for next patches.